### ISSUE TITLE
SRTS and Rimthreaded WorldTargeter problem

### ISSUE THREAD

**Issue by alighahremani97** [cseelhoff/RimThreaded#531]
**Describe the bug**
SRTS could not launch any ships, in addition, the vanilla pods would not launch either. I could launch SRTS ships on my own tile, but anything from one tile away to many tiles away was bugged (SRTS would ping me on the world map that it needs 2.1 billion chemfuel, underflow of a 4 byte int variable). Upon removing rimthreaded's latest version (as per discord release), the issue was fixed. One of the mod developers of SRTS kindly helped me with this, there are also 4 transpilers on the world alone according to him.

**Steps to reproduce the behavior (VERY IMPORTANT)**
1. Click the "pelican" ship (giant freaking ship by the water)
2. Click on Launch (pilot already in there)
3. On the world map, any tile apart from my on base's tile would give you the error
4. See error on the top left

**Error Log**
https://gist.github.com/fa06f6d45d8db5e63af64f566101bb1e

**Save file**
[Terran Expedition.zip](https://github.com/cseelhoff/RimThreaded/files/6603464/Terran.Expedition.zip)

**Screenshots**
none

**Mod list (Preferably a RimPy compatible list.)**
[rimthreaded bug upload.txt](https://github.com/cseelhoff/RimThreaded/files/6603465/rimthreaded.bug.upload.txt)

**Comment by pastorismylord** [comment]
you're using incompatible mods with that version of rimthreaded. Either remove the incompatible mods or use a preview version of rimthreaded.

**Comment by alighahremani97** [comment]
I am using the preview released days ago. Also I don't understand, your incompatibility list is so huge and changing all the time that it makes no sense to me. It appears that most of these incompatibilities are based on votes? most of the mods I have also have description on your github page for incompatibilities that are described as "certain issues" like "locking" or "freezing" or buggy interactions. nothing like what I described above. But ok, since you want to just close it without telling me which mods were incompatible, its better that I switch this off entirely. It has caused more harm than good so far for me. Rocketman seems infinitely better

**Comment by alighahremani97** [comment]
Also did you even bother reading what I posted? the SRTS developer said that its a WorldTargeter issue. Which ALSO effects fire missions for the artillery pieces of Rimatomics, as well as vanilla transport pod. I wasn't using this mod anyways I thought it would be helpful to let you know what the issue was rather than just unsub and leave. But seeing your attitude its better that I just leave

**Comment by pastorismylord** [comment]
the logs and mod list do not include rimthreaded.

**Comment by alighahremani97** [comment]
See now that could have been said, which is an issue on my end, but I uninstalled it so I could get the save going, and probably forgot to include back in RimPy's. The issue is the same as the one you just mentioned. but I doubt an average user is going to comb through 100s of posts for such mentions (despite the fact that I tried and missed that one, because I was solely focused on SRTS). Good luck to you folks with multithreading this, but the mod just isnt for me
